packages/node-cli/src/commands/account/savetoken/AccountSaveTokenAction.js

```diff
@@ -7,7 +7,7 @@
 const path = require('path');
 const BaseAction = require('../../base/BaseAction');
 const { saveToken } = require('../../../utils/AuthenticationUtils');
-const { PROD_ENVIRONMENT_ADDRESS } = require('../../../ApplicationConstants');
+const { DOMAIN } = require('../../../ApplicationConstants');
```

Comment:
Could we import only the message used? DOMAIN.PRODUCTION.GENERIC_NETSUITE_DOMAIN

Reply:
Sure!